src/main/resources/templates/editPassword.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <title>Change Password</title>
+</head>
+<body>
+<div id="nav-placeholder"></div>
+<div class="login-container">
+    <form action="#" th:action="@{/editPassword}" method="post"
+          class="container d-flex flex-column flex-md-col col-sm-12 col-md-6 col-lg-4">
+        <h3>change your password</h3><br>
+        <div class="form-group">
+            <label>Enter Your Password:</label>
+            <input type="password" class="form-control"
+                   name="oldPassword" id="oldPassword"
+                   required placeholder="required"/>
+        </div>
+        <div class="password_match" data-msg="Password doesn't match.">
+            <div class="password form-group">
+                <label>Enter New Password:</label>
+                <input type="password" name="newPassword"
+                       id="password" class="form-control"
+                       required placeholder="required"
+                       pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+                       title="8 Characters: 1 Upper, 1 Lower, 1 Digit, 1 Special Characters. Must Contain!!!">
```

Comment:
Maybe write that in a normal sentence? i.e. "Password must be 8 characters long and contain: 2 digit, 1 uppercase, 1 lowercase & 1 special character"